spring-cloud-dataflow-server-core/src/main/java/org/springframework/cloud/dataflow/server/stream/SkipperStreamDeployer.java

```diff
@@ -184,18 +192,23 @@ public void deployStream(StreamDeploymentRequest streamDeploymentRequest) {
 		// Install the package
 		InstallRequest installRequest = new InstallRequest();
 		PackageIdentifier packageIdentifier = new PackageIdentifier();
-		packageIdentifier.setPackageName(streamDeploymentRequest.getStreamName());
+		String streamName = streamDeploymentRequest.getStreamName();
+		packageIdentifier.setPackageName(streamName);
 		packageIdentifier.setPackageVersion("1.0.0");
-		packageIdentifier.setRepositoryName("local");
+		String repoName = "local";
+		packageIdentifier.setRepositoryName(repoName);
 		installRequest.setPackageIdentifier(packageIdentifier);
 		InstallProperties installProperties = new InstallProperties();
 		installProperties.setPlatformName("default");
 		//todo: Better naming for the release name
-		installProperties.setReleaseName("my" + streamDeploymentRequest.getStreamName());
+		String releaseName = "my" + streamName;
```

Comment:
Should `my` be something like `SCDF`?

Reply:
We have a story to cover this: https://github.com/spring-cloud/spring-cloud-dataflow/issues/1688